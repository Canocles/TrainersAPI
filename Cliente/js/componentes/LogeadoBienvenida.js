import React from 'react'

class LogeadoBienvenida extends React.Component {
    render() {
        return (
            <div className="container" style={{ padding: 10 }}>
                <div className="jumbotron">
                    <h1>Bienvenido a nuestra plataforma</h1>
                </div>
                <div className="alert alert-danger fade-in">
                    <a href="javascript:void(null)" className="close" data-dismiss="alert" aria-label="close">&times;</a>
                    <strong>Atención:</strong> Esta aplicación está en construcción, por lo que únicamente podrá interactuar con sus entrenamientos.
                </div>
                <h3>
                    En nuestra plataforma, vas a poder crear, modificar o eliminar entrenamientos para de esta forma
                    seguir con un plan de entrenamiento.
                </h3>
            </div>
        )
    }
}

export default LogeadoBienvenida;