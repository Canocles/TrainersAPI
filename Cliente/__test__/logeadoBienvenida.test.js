import React from 'react';
import renderer from 'react-test-renderer'
import LogeadoBienvenida from '../src/componentes/LogeadoBienvenida';

describe('LogeadoBienvenida (snpashot)', () => {
  it('El componente de LogeadoBienvenida se renderiza correctamente', () => {
    const component = renderer.create(<LogeadoBienvenida />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
