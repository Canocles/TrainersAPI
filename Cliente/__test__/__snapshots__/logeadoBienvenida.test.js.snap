// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogeadoBienvenida (snpashot) El componente de LogeadoBienvenida se renderiza correctamente 1`] = `
<div
  className="container"
  style={
    Object {
      "padding": 10,
    }
  }
>
  <div
    className="jumbotron"
  >
    <h1>
      Bienvenido a nuestra plataforma
    </h1>
  </div>
  <div
    className="alert alert-danger fade-in"
  >
    <button
      aria-label="close"
      className="close"
      data-dismiss="alert"
      href="#"
    >
      ×
    </button>
    <strong>
      Atención:
    </strong>
     Esta aplicación está en construcción, por lo que únicamente podrá interactuar con sus entrenamientos.
  </div>
  <h3>
    En nuestra plataforma, vas a poder crear, modificar o eliminar entrenamientos para de esta forma seguir con un plan de entrenamiento.
  </h3>
</div>
`;
